instrumentation/grpc/src/main/java/brave/grpc/TracingClientInterceptor.java

```diff
@@ -117,9 +90,14 @@ void finishWithError(@Nullable Span span, Throwable error) {
       try (Scope scope = currentTraceContext.maybeScope(span.context())) {
         super.start(responseListener, headers);
       } catch (Throwable e) {
+        propagateIfFatal(e);
+
         // Another interceptor may throw an exception during start, in which case no other
         // callbacks are called, so go ahead and close the span here.
-        finishWithError(spanRef.getAndSet(null), e);
+        //
+        // See instrumentation/grpc/RATIONALE.md for why we don't use the handler here
+        spanRef.getAndSet(null);
```

Comment:
Either just `set` or pass the return value to the next line.

Reply:
set is better yeah. thanks